coretex/cli/modules/project_utils.py

```diff
@@ -55,3 +56,16 @@ def getProject(name: Optional[str], config: Dict[str, Any]) -> Optional[Project]
         "Project can be selected globally by using \"coretex project select\" command\n"
         "or you can pass the Project directly to this command using \"--project\" option"
     )
+
+
+def isProjectSelected() -> bool:
+    config = loadConfig()
+
+    if config.get("projectId") is None:
+        return False
+
+    try:
+        Project.fetchById(config["projectId"])
+        return True
+    except:
```

Comment:
Handle the appropriate exception which is raised if `Project.fetchById` call fails